client/src/Components/Layouts/Order/addOrder.jsx

```diff
@@ -0,0 +1,418 @@
+                className="userName"
+              >
+                {getFieldDecorator("userName", {
+                  rules: [
+                    {
+                      required: true,
+                      message: "يرجى ملئ الحقل ",
+                      pattern: /^([أ-يa-z0-9]|\s)*$/
+                    }
+                  ]
+                })(
+                  <div>
+                    <Input />
+                  </div>
+                )}
+              </Form.Item>
+              <Form.Item label="عنوان الزبون" className="address">
+                {getFieldDecorator("address", {
+                  rules: [
+                    {
+                      required: true,
+                      message: "يرجى ملئ الحقل ",
+                      pattern: /^([أ-يa-z0-9-]|\s)*$/
+                    }
+                  ]
+                })(<Input />)}
+              </Form.Item>
+              <div style={{ display: "flex" }}>
+                <Form.Item label="الهاتف" layout="horizontal" className="phone">
+                  {getFieldDecorator("phone", {
+                    rules: [
+                      {
+                        required: true,
+                        message: "يرجى مليء الحقل بتسعة ارقام",
+                        pattern: /^([0-9]){9}$/
+                      }
+                    ]
+                  })(<Input />)}
+                </Form.Item>
+                <Form.Item
+                  layout="horizontal"
+                  style={{ direction: "rtl", flexDirection: "row-reverse" }}
+                >
+                  {getFieldDecorator("phone1", { initialValue: "970" })(
+                    <Select
+                      style={{
+                        direction: "ltr",
+                        width: "83px",
+                        marginRight: "-6px"
+                      }}
+                    >
+                      <Option value="970">+970</Option>
+                      <Option value="972">+972</Option>
+                    </Select>
+                  )}
+                </Form.Item>
+              </div>
+              <Form.Item
+                label="اسم الكابتن"
+                layout="horizontal"
+                className="captainName"
+              >
+                <AutoComplete
+                  className="certain-category-search"
+                  dropdownClassName="certain-category-search-dropdown"
+                  dropdownMatchSelectWidth={false}
+                  dropdownStyle={{ width: 300 }}
+                  size="large"
+                  style={{ width: "100%" }}
+                  dataSource={optionsCaptains}
+                  placeholder="اختر الكابتن"
+                  optionLabelProp="value"
+                  onBlur={this.handleBlureCaptain}
+                  filterOption={(inputValue, option) =>
+                    option.props.children
+                      .toUpperCase()
+                      .indexOf(inputValue.toUpperCase()) !== -1
```

Comment:
@denaHS I usually also trim() the inputValue when filtering like this, so if the user accidentally has a space at the beginning or end of their input, it still will work (this can happen if they copy-paste input into the field)